docs/resource/host.md

```diff
@@ -16,6 +16,9 @@
 | `host.image.name` | string | Name of the VM image or OS install the host was instantiated from. | `infra-ami-eks-worker-node-7d4ec78312`; `CentOS-8-x86_64-1905` | Recommended |
 | `host.image.id` | string | VM image ID or host OS image ID. For Cloud, this value is from the provider. | `ami-07b06b442921831e5` | Recommended |
 | `host.image.version` | string | The version string of the VM image or host OS as defined in [Version Attributes](README.md#version-attributes). | `0.1` | Recommended |
+| `host.mac` | string[] | Available MAC addresses of the host, excluding loopback interfaces. [1] | `[ac-de-48-23-45-67, AC-DE-48-23-45-67-01-9F]` | Opt-In |
+
+**[1]:** MAC Addresses MUST be represented in [IEEE RA hexadecimal form](https://standards.ieee.org/wp-content/uploads/import/documents/tutorials/eui.pdf): as hyphen-separated octets in hexadecimal form from most to least significant.
```

Comment:
Not sure if this should be a MUST; hyphens seems to be the IEEE standard but I don't think it is unusual to see MAC addresses separated by colons (e.g. Go will separate by `:`, see https://cs.opensource.google/go/go/+/refs/tags/go1.21.1:src/net/mac.go;l=19). Maybe it should be a SHOULD instead? Or maybe we should abandon IEEE RA notation and do "IEEE RA notation but with colons"?

Reply:
There was a similar discussion for ECS at https://github.com/elastic/ecs/pull/456#issuecomment-767099447. This comment mostly sums it up how it was decided.

> There are two competing standards. Lets just go with one of them. Better than having implementers choose one at random.

I think a stricter definition will be better for consistency, and this predictability is good if you are trying to correlate or search. Toward that goal of consistency in the data, I think the definition should advise either lowercase or uppercase.

(new here, so if this goes against some core principles please let me know, thanks)

Reply:
The ECS conclusion sounds reasonable to me. I did not specify casing intentionally to give some leeway to implementations, but I guess it's better to be stricter; we can always relax the condition if it seems impractical in the future.

Reply:
Changed, thanks for the suggestion and the context :)